std_msgs/Header header
uint32 id
float32 pos_x
float32 pos_y
float32 pos_z
float32 att_roll
float32 att_pitch
float32 att_yaw
float32 dx
float32 dy
float32 dz
float32 d_yaw
float32 vel_ff_x
float32 vel_ff_y
float32 vel_ff_z
uint32 type
uint32[5] flag
float32 send_currentpos_freq
float32 send_desirepos_freq
float32 send_type_freq
float32 q1
float32 q2
float32 q3
float32 q4